src/core/thread/router_table.cpp

```diff
@@ -550,6 +550,47 @@ Error RouterTable::SetRouterIdRange(uint8_t aMinRouterId, uint8_t aMaxRouterId)
 }
 #endif
 
+#if OT_SHOULD_LOG_AT(OT_LOG_LEVEL_INFO)
+void RouterTable::LogRouteTable(void)
+{
+    static constexpr uint16_t kStringSize = 128;
+
+    LogInfo("Route table");
+
+    for (Router &router : Iterate())
+    {
+        String<kStringSize> string;
+
+        string.Append("    %2d 0x%04x", router.GetRouterId(), router.GetRloc16());
+
+        if (router.GetRloc16() == Get<Mle::Mle>().GetRloc16())
+        {
+            string.Append(" - me");
+        }
+        else
+        {
+            if (router.IsStateValid())
+            {
+                string.Append(" - neigh{lq[i/o]:%d/%d cost:%d}", router.GetLinkQualityIn(), router.GetLinkQualityOut(),
```

Comment:
nit: what about `nbr` as a short abbreviation?
```suggestion
                string.Append(" - nbr{lq[i/o]:%d/%d cost:%d}", router.GetLinkQualityIn(), router.GetLinkQualityOut(),
```

Reply:
Good suggestion. Thanks. Changed in new push.